Beta

prj84 opened the pull request.

svendberg commented:
Nice!

I have also been poking on ZigBeeDevice.js trying to work around this problem with no real luck. This "fix" will at least stop the app from living a life of it own when the same light is adjusted too many times. 

The only problems i see: 
1. As you say, the fix is in node_modules and will be a pain to maintain to prevent it from being overwritten.
2. I guess this will affect the thermostats as well since the plugin supports both the ESH dimmer and thermostat.

Espenhh commented:
Do you have any thoughts about this PR @prj84?

Regarding your problems @svendberg:

1. I feel like this is something one can live with, the worst thing that happens if the node-module is updated is that we return to how it is today. 
2. Affecting the termostats: I didn't think about that. To be honest, I haven't made much use of "intelligent control" of the termostats yet…

prj84 commented:
Hi!
I'm sorry that this has taken so long! Work, holiday, kids anda family takes up nearly all my time these days. I have been looking at your code and it seems to be the best way to deal with this at the moment. It will affect the thermostats as well. I have automated mine, but this is just one command twice a day, so can not see big issues with this. Maybe it will cause some problems when using the "thermostat weel" in the app. But lets go!

prj84 commented:
Just sent the beta 0.0.4 to review including this fix. So should hopefully be in the app store in a few days.

Espenhh commented:
Yay, great! I've been running this "version" on my system for a few weeks now, it has worked quite well. So at least the time gave me some proper testing periode (totally understand that life takes over, knows how this is myself, so no worries! 😊)

PS: do you know any good lines in to the Homey people? I would like to try nudging them a bit more on the root issue here myself, so if you have any tips on which channel is best suited let me know.

prj84 committed:
Fixed kWh bug, return to old logic.
Added SmartStikkDobbel & SmartStikkEnkel
Added all Multiwire dimmer id's to code
Made preperations for PIR devices

prj84 committed:
Bump version to v1.1.0

Changelog: Fixed kWh bug, returned to old logic. Added SmartStikkDobbel & SmartStikkEnkel. Added Multiwire dimmer id's to code. Made preperations for PIR devices.
